cmd/check-reaper/check-reaper.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: check-reaper
+  namespace: kuberhealthy
+spec:
+  schedule: "*/3 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: check-reaper
+              image: quay.io/comcast/check-reaper:1.0.0
+              imagePullPolicy: Always
```

Comment:
change imagepullpolicy to IfNotPresent?

Reply:
this runs every 3 mins, so best not to pull it so often

Reply:
✅